MeetU/MeetU/Views/Account/Login.cshtml

```diff
@@ -48,11 +48,18 @@
 @section Scripts {
     @Scripts.Render("~/bundles/jqueryval")
     <script>
-        $(".js-login-jump-icon").click(function () {
+        if ($(".validation-summary-errors").length > 0) {
+            scrollLogin(0)
+        } else {
+            setTimeout(scrollLogin, 1500, 900);
+        }
+
+        // Scroll down to login panel
+        function scrollLogin(speed) {
```

Comment:
you can set a default value to speed, so that if you call scrollLogin() without parameter specified, the speed will be the default value 0.